local-volume/README.md

```diff
@@ -165,30 +165,39 @@ $ kubectl create -f bootstrapper/deployment/kubernetes/example-storageclass.yaml
 
 ### Step 3: Creating local persistent volumes
 
-#### Option 1: Bootstrapping the external static provisioner
+#### Option 1: Using the local volume static provisioner 
 
-This is optional, only for automated creation and cleanup of local volumes. See
-[bootstrapper/](./bootstrapper) and [provisioner/](./provisioner) for details and
-sample configuration files.
+	1. Install helm
+See README.md located in ./local-volume/helm directory.
```

Comment:
I think you can remove the "local-volume" prefix to all these paths because this README is already inside that directory.

Reply:
done